README.md

```diff
@@ -196,6 +204,12 @@ export default createPlugin({
       return next();
     };
   },
+  // dependency injected async cleanup function
```

Comment:
nit: "dependency injected" seems somewhat confusing to me given only `provided`(`thing`) is available